data/campaigns/Delfadors_Memoirs/scenarios/12_Terror_at_the_Ford_of_Parthyn.cfg

```diff
@@ -350,10 +350,16 @@
         first_time_only=yes
         [filter]
             side=1
-            x={X}
-            y={Y}
-            radius={RADIUS}
         [/filter]
+        [filter_condition]
+            [have_location]
+                x,y=$x1,$y1
+                [and]
+                    x,y={X},{Y}
+                    radius={RADIUS}
+                [/and]
+            [/have_location]
+        [/filter_condition]
```

Comment:
This can be written in a shorter way as [filter_location] x,y,radius={X},{Y},{RADIUS} [/filter_location] and included directly in the [filter] block. The original was simply missing the [filter_location] tag around those keys.